Packs/Active_Directory_Query/Playbooks/Active_Directory_-_Get_User_Manager_Email_README.md

```diff
@@ -33,4 +33,4 @@ This playbook does not use any sub-playbooks.
 
```

Comment:
I saw in the playbook yml you used extandcontext to the manager details - we should change the playbook outputs accordingly (I think task 7)

Reply:
The outputs are set later on in the outputs. The extend-context is only used to differentiate between the key that holds the user details and the key that holds the manager details (we get both of them from AD)